src/applications/simple-forms/21-0966/containers/ConfirmationPage.jsx

```diff
@@ -21,170 +21,159 @@ import {
-              <p>{confirmationNumber}</p>
-            </>
-          )}
+  const dateOptions = {
+    weekday: 'long',
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  };
+  const expirationDate = new Date(
+    submission.response?.expirationDate,
+  ).toLocaleDateString('en-US', dateOptions);
+  const alreadySubmittedIntents = {};
+  if (submission.response?.compensationIntent) {
+    alreadySubmittedIntents.compensation =
+      submission.response.compensationIntent;
+  }
+  if (submission.response?.pensionIntent) {
+    alreadySubmittedIntents.pension = submission.response.pensionIntent;
+  }
+  if (submission.response?.survivorIntent) {
+    alreadySubmittedIntents.survivor = submission.response.survivorIntent;
+  }
 
-          {isValid(submitDate) && (
-            <>
-              <h4>Date submitted</h4>
-              <p>{format(submitDate, 'MMMM d, yyyy')}</p>
-            </>
-          )}
+  const alreadySubmittedTitle = getAlreadySubmittedTitle(
+    data,
+    alreadySubmittedIntents,
+  );
+  const alreadySubmittedText = getAlreadySubmittedText(
+    data,
+    alreadySubmittedIntents,
+    expirationDate,
+  );
+  const nextStepsTextSecondParagraph = getNextStepsTextSecondParagraph(
+    data,
+    alreadySubmittedIntents,
+    expirationDate,
+  );
+  const nextStepsLinks = getNextStepsLinks(data);
 
-          <h4>Confirmation for your records</h4>
-          <p>You can print this confirmation page for your records</p>
-          <button
-            type="button"
-            className="usa-button vads-u-margin-top--0 screen-only"
-            onClick={window.print}
-          >
-            Print this page
-          </button>
-        </div>
-        {alreadySubmittedTitle && alreadySubmittedText ? (
-          <div>
-            <h2>{alreadySubmittedTitle}</h2>
-            <p>{alreadySubmittedText}</p>
-          </div>
-        ) : null}
-        <div>
-          <h2>What are my next steps?</h2>
-          <p>You should complete and file your claim as soon as possible.</p>
-          <p>{nextStepsTextSecondParagraph}</p>
-          {nextStepsLinks.map(nextStep => {
-            let href = '/';
-            if (nextStep === veteranBenefits.COMPENSATION) {
-              href =
-                '/disability/file-disability-claim-form-21-526ez/introduction';
-            } else if (nextStep === veteranBenefits.PENSION) {
-              href = '/find-forms/about-form-21p-527ez/';
-            } else if (nextStep === veteranBenefits.SURVIVOR) {
-              href = '/find-forms/about-form-21p-534ez/';
-            }
+  const alertType = getAlertType(data, alreadySubmittedIntents);
+  const alertTitle =
+    alertType === 'info'
+      ? getInfoAlertTitle()
+      : getSuccessAlertTitle(data, alreadySubmittedIntents);
+  const alertText =
+    alertType === 'info'
+      ? getInfoAlertText(data, alreadySubmittedIntents)
+      : getSuccessAlertText(data, alreadySubmittedIntents);
```

Comment:
refactored this part. not strictly necessary, but I had already done it while trying to triage the issue.